src/loader/bgw_launcher.c

```diff
@@ -555,26 +557,24 @@ message_stop_action(HTAB *db_htab, BgwMessage *message)
 }
 
 /*
- * This function will only restart an existing scheduler and will throw an error
- * if it is told to restart a nonexistent or disabled scheduler.
- *
- * One might think that this function would simply be a combination of stop and start above, however
- * we decided against that because we want to maintain the worker's "slot".
- * We don't want a race condition where some other db steals the scheduler of the other by requesting a worker at the wrong time.
+ * This function will restart an existing scheduler and start a scheduler if it
```

Comment:
What does it mean for a scheduler to "exist"? Does "existing" mean "running" here or just that there is a DB entry in the hash table? It is not clear whether a scheduler could "exist" but not be running. 

Renaming `DbHashEntry` to something like `SchedulerEntry` (or just `Scheduler`) would perhaps clarify the semantics of that entry? That would make it more clear what it means for a scheduler to exist?